com.parallax.PropellerIDE.json

```diff
@@ -0,0 +1,170 @@
+                    "branch": "master"
+                }
+            ]
+        },
+        {
+            "name": "library",
+            "buildsystem": "simple",
+            "build-commands": [
+                "find ./library -type f -exec install -D '{}' '/app/share/propelleride/library/{}' \\;"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/parallaxinc/spin-standard-library.git",
+                    "branch": "master"
+                }
+            ]
+        },
+        {
+            "name": "doc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "find . -type f -exec install -D '{}' '/app/share/propelleride/doc/{}' \\;"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/parallaxinc/spin-docs.git",
+                    "branch": "master"
+                }
+            ]
+        },
+        {
+            "name": "libusb",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/libusb/libusb.git",
+                    "tag": "v1.0.23"
+                }
+            ]
+        },
+        {
+            "name": "libconfuse",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/libconfuse/libconfuse.git",
+                    "tag": "v3.3"
+                }
+            ]
+        },
+        {
+            "name": "libftdi",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://developer.intra2net.com/libftdi",
+                    "tag": "v1.5"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 com.parallax.PropellerIDE.desktop /app/share/applications/com.parallax.PropellerIDE.desktop",
+                "install -Dm644 com.parallax.PropellerIDE.svg /app/share/icons/hicolor/scalable/apps/com.parallax.PropellerIDE.svg",
+                "install -Dm544 com.parallax.PropellerIDE.appdata.xml /app/share/appdata/com.parallax.PropellerIDE.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "com.parallax.PropellerIDE.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.parallax.PropellerIDE.svg"
+                },
+                {
+                    "type": "file",
+                    "path": "com.parallax.PropellerIDE.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html